scripts/onboarding/aks/onboarding-using-msi-auth/existingClusterOnboarding.json

```diff
@@ -80,8 +80,7 @@
         "clusterResourceGroup": "[split(parameters('aksResourceId'),'/')[4]]",
         "clusterName": "[split(parameters('aksResourceId'),'/')[8]]",
         "clusterLocation": "[replace(parameters('aksResourceLocation'),' ', '')]",
-        "workspaceLocation":"[replace(parameters('workspaceRegion'),' ', '')]",
-        "dcrNameFull": "[Concat('MSCI', '-', variables('workspaceLocation'), '-', variables('clusterName'))]",
+        "dcrNameFull": "[Concat('MSCI', '-', parameters('workspaceRegion'), '-', variables('clusterName'))]",
```

Comment:
The reason for having  workspaceLocation parameter to trim the whitespaces. looks like removing this variable which can break if the workspace region has spaces.  Lets keep the change, what we have.

Reply:
reverted